l2-contracts/contracts/bridge/L2ERC20Bridge.sol

```diff
@@ -28,6 +28,8 @@ contract L2ERC20Bridge is IL2Bridge, Initializable {
     /// @dev Bytecode hash of the proxy for tokens deployed by the bridge.
     bytes32 internal l2TokenProxyBytecodeHash;
 
+    address public governor;
```

Comment:
What is it used for?

Reply:
I think we might need it in the longer term, but I will just remove it then for now